config/configuration_singleton.rb

```diff
@@ -109,6 +109,25 @@ def quota_threshold
     ENV.fetch("OOD_QUOTA_THRESHOLD", 0.95).to_f
   end
 
+  # The paths to the JSON files that store the balance information
+  # Can be URL or File path. colon delimited string; though colon in URL is
+  # ignored if URL has format: scheme://path (colons preceeding // are ignored)
+  #
+  # /path/to/balance.json:https://osc.edu/balance.json
+  #
+  #
+  # @return [Array<String>] balance paths
+  def balance_paths
+    # regex uses negative lookahead to ignore : preceeding //
+    ENV.fetch("OOD_BALANCE_PATH", "").strip.split(/:(?!\/\/)/)
+  end
+
+  # The threshold for determining if there is sufficient balance remaining
+  # @return [Float] threshold factor
+  def balance_threshold
+    ENV.fetch("OOD_BALANCE_THRESHOLD", 0).to_f
```

Comment:
The default threshold should be higher than 0. If 0, the warning message will always display, if this works like the quota warnings do.

Reply:
Sorry I realize now that the threshold value is an integer i.e. number of RUs, rather than a percentage. Markign this as resolved.